core/src/main/scala/org/apache/spark/deploy/DeployMessage.scala

```diff
@@ -108,6 +108,9 @@ private[deploy] object DeployMessages {
 
   case class Heartbeat(workerId: String, worker: RpcEndpointRef) extends DeployMessage
 
+  // Out of band commands to Master
+  case class DecommissionHosts(hostnames: Seq[String])
```

Comment:
If we want to be accurate, this should be `DecommissionWorkersByHost`.

Looking at other classes in this file, they are all about master/worker, e.g. `RegisterWorker`, `WorkerDecommission`, etc. That's why I suggest `DecommissionWorkers`, and add comments to explain the workers are specified by hosts.

Reply:
Makes sense !. I like DecommissionWorkersByHost.